scripts/attention_tests.py

```diff
@@ -0,0 +1,319 @@
+import torch
```

Comment:
Can you move it to scripts/tests/attention.py

Reply:
Done.